internal/controller/drplacementcontrol.go

```diff
@@ -2298,7 +2298,7 @@ func updateDRPCProgression(
 		// caller of this function is always d.setProgression()
 		// caller of d.setProgression() makes the progression decision.
 		// Use ancestorLevel=2 to get the caller of the caller.
-		// nolint: gomnd
+		// nolint: mnd
```

Comment:
This is here so we don't have to backport another (unrelated to the cve) commit?

Reply:
The backport [here](https://github.com/red-hat-storage/ramen/pull/460/commits/336a019fda04bad038bfed82a4e46e461837509d) changed this to the older scheme, the new ci scheme is mnd and required a further update as per upstream.